netlify.toml

```diff
@@ -11,7 +11,7 @@
   REACT_APP_GA_MEASUREMENT_ID="G-6ZGJM8JXYN"
   REACT_APP_TOUCAN_LINK="https://toucan.earth/overview?chainId=137"
 
-[context.deploy-preview.environment]
+[context.dev.environment]
```

Comment:
we still want to keep the context for deploy-previews I think or is the plan just that they are going to use the default envs set up from netlify UI?

Reply:
This is a mistake, this entry should still exist yes, will fix that thanks.